react/components/Metrics/DataAnalysis/CustomElements/CustomXAxisTick.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+
+interface Props {
+  x: number
+  y: number
+  payload: any
+}
+
+
+const CustomXAxisTick = ({ x, y, payload }: Partial<Props>) => (
```

Comment:
Why are you using `Partial` here?

(You can use `React.SFC` to declare the type here.)